src/test/java/duketest/TodoTest.java

```diff
@@ -0,0 +1,14 @@
+package duketest;
+
+import duke.Todo;
+
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+public class TodoTest {
```

Comment:
I think I'm just being paranoid, but I prefer having an empty line between the imports and the class declaration.